apps/web/src/app/(signing)/sign/[token]/form.tsx

```diff
@@ -140,6 +142,7 @@ export const SigningForm = ({ document, recipient, fields }: SigningFormProps) =
                 isSubmitting={isSubmitting}
                 onSignatureComplete={handleSubmit(onFormSubmit)}
                 document={document}
+                enableTips={enableTips}
```

Comment:
It's okay to have simple inline handlers like

```
enableTips={() => setValidateUninsertedFields(true)}
```

I'd also suggest refactoring `tips` to `tooltips`